client/src/utils.ts

```diff
@@ -126,3 +126,13 @@ export const getSubtotalPrice = (cartItem: CartItem) => {
     PRICE_FRACTION_DIGITS
   );
 };
+
+// Returns a list of strings
+export const parseRawTextNewlines = (text: string): string[] => {
+  return text.split("%0A");
+};
+
+// Returns formatted address seperated into lines
+export const parseRawAddressNewlines = (addr: string): string[] => {
+  return addr.split(",");
```

Comment:
Address can be like  ```#01-06, Somehting somethign, Somewhere, On some road.``` is there a way to determine the original lines? This formatt lines like "#01-06,
Something Something,
Somewhere
On Some Road ```

Reply:
hmm, so to rephrase the question:
given a poorly formatted address: `#01-06, Somehting somethign, Somewhere, On some road.`, how can we automatically do error-correction and turn `#01-06 Somehting somethign Somewhere On some road.` (note: I've taken out the potentially wrongly-placed commas) into:
```
Something somethign #01-06
Somewhere
On some road.
```

Reply:
In this case, it is not a formatting task but a NLP task, we would need a tokenizer + parser for the different address formats potentially by region? (like US has different way of breaking address into lines than SG + additional required State information)
Currently, I'm just pulling the `vicinity` field from Places API and breaking lines by commas, however, I see that on Google Maps, they display this `vicinity` string as a whole without breaking by commas, maybe we can also adopt this and not break into multiple lines

Reply:
May out of context of this cl:
so you get this address from the db right? Perhaps the address storage needs to be split into lines like: (maybe 3 is enough, max 4)
Address: {
Address line 1: string,
Address line 2: string,
Address line 3: string,
Pincode: number
city/state/country
}

Reply:
I see, currently the NearbyPlaces lookup does not return data to this specificity. However, a subsequent call to Place Detail API may be needed anyway to determine if a location is currently open. In this response we will have a pre-formatted "formatted_address" which we can pull to display instead

Reply:
to address in #161